sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -384,5 +403,7 @@ def update(self):
         dynamodb_client.get_waiter("table_exists").wait(TableName=f"{self.name}")
 
     def teardown(self):
-        dynamodb_resource = _initialize_dynamodb_resource(region=self.region)
+        dynamodb_resource = _initialize_dynamodb_resource(
+            region=self.region, endpoint_url=None
```

Comment:
same question

Reply:
I've made some changes based on your suggestions, if you don't mind giving it a second check, please.